tests/integration/test_overview.py

```diff
@@ -17,18 +17,6 @@ def test_overview_type_changes_correctly(base_url, selenium):
     assert exp_type in experiment.experiment_type
 
 
-@pytest.mark.nondestructive
-def test_overview_owner_changes_correctly(base_url, selenium):
```

Comment:
Instead of removing this can we just skip it? ```@pytest.mark.skip``` should do the trick.

Reply:
Good idea!